### PR TITLE
Categorized each pizza into a type

### PR DESCRIPTION
**Pull request by upulieh**
Types - Classic , Signature, Supreme

### CODE REVIEW

File: frontend/src/components/PizzaList.js [hunk 1]
```diff
@@ -1,22 +1,70 @@
 import Pizza from "./Pizza";
 import { Container, Row, Col } from "react-bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { useState, useEffect } from "react";
 
 const PizzaList = (props) => {
   const listOfPizzas = props.listOfPizzas;
   const onAdd = props.onAdd;
+  const categories = ["classic", "supreme", "signature"]; //try thiis
```
Comment (vongohren) [inline frontend/src/components/PizzaList.js:9]: Are these your pizzas or just types of Pizzas?

File: frontend/src/components/PizzaList.js [hunk 2]
```diff
@@ -1,22 +1,70 @@
 import Pizza from "./Pizza";
 import { Container, Row, Col } from "react-bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { useState, useEffect } from "react";
 
 const PizzaList = (props) => {
   const listOfPizzas = props.listOfPizzas;
```
Comment (vongohren) [inline frontend/src/components/PizzaList.js:7]: I have not seen this list, want to send a link to the place this is?

Comment (upulieh) [reply frontend/src/components/PizzaList.js:7]: I sent you an invite to join the project. Please accept.

File: frontend/src/components/PizzaList.js (hunk 1 above)
Comment (upulieh) [reply frontend/src/components/PizzaList.js:9]: These are types of pizzas. Each pizza is categorized into either of the three types.

File: frontend/src/components/PizzaList.js (hunk 2 above)
Comment (vongohren) [reply frontend/src/components/PizzaList.js:7]: @upulieh I have not seen it? Neither by email nor notifications

File: frontend/src/components/PizzaList.js (hunk 1 above)
Comment (vongohren) [reply frontend/src/components/PizzaList.js:9]: Thanks!

**Pull request merged**